jetpack.php

```diff
@@ -133,6 +133,9 @@ function jetpack_admin_unsupported_php_notice() { ?>
 	return;
 }
 
+// Load all our packages.
```

Comment:
could be other packages in the future too :)

Reply:
Great point, corrected in https://github.com/Automattic/jetpack/pull/12379/commits/3d047e2c4dd4b41f03502499f7a3d31221250fac